public/quickstart.js

```diff
@@ -1,9 +1,34 @@
-var videoClient;
 var activeRoom;
-var previewMedia;
+var previewTracks;
 var identity;
 var roomName;
 
+function attachMedia(tracks, container) {
+  tracks.map(function(track) {
+    return track.attach();
+  }).forEach(function(mediaElement) {
+    container.appendChild(mediaElement);
+  });
+}
+
+function attachParticipantMedia(participant, container) {
+  attachMedia(Array.from(participant.tracks.values()), container);
+}
+
+function detachMedia(tracks) {
+  tracks.map(function(track) {
+    return track.detach();
+  }).forEach(function(mediaElements) {
+    mediaElements.forEach(function(mediaElement) {
+      mediaElement.remove();
+    });
```

Comment:
Instead of two iterations of the loop in `attachMedia` and `detachMedia`, can we do a single?